### ISSUE TITLE
enhancement to existing items

### ISSUE THREAD

**Issue by sanuann** [ReproNim/demo-protocol#9]
- [ ] multi-select from a dropdown(the 4th item in the activity). the existing item advances when a single option is selected. this needs to be fixed such that first, you choose as many items you wish from the dropdown and then hit a submit button to advance to the next question.
- [ ] similar for the 6th item
- [ ] slider will need to be fixed or make a new slider component if fix not possible

**Comment by sanuann** [comment]
@MulliganF02 this is assigned to you.

**Comment by Remi-Gau** [comment]
There also seems to be an issue on the `button_start_item`. :wink: 

![Screenshot 2021-02-26 09:54:51](https://user-images.githubusercontent.com/6961185/109278357-bbc91d80-7818-11eb-83d8-df8b2064ce1c.png)

**Comment by sanuann** [comment]
> There also seems to be an issue on the `button_start_item`. 😉
> 
> ![Screenshot 2021-02-26 09:54:51](https://user-images.githubusercontent.com/6961185/109278357-bbc91d80-7818-11eb-83d8-df8b2064ce1c.png)

yes, I've seen that. I dunno what a button_start_item means. I think that is an error and needs to be removed. Will look into it.

**Comment by Remi-Gau** [comment]
No rush on my side: I was just going through all the items just to see. :smile: 

Keep up the awesome work. 

# :pray: :rocket:
